.gn

```diff
@@ -18,7 +18,14 @@ secondary_source = "//flutter/build/secondary/"
 check_targets = [
```

Comment:
Should we take the opportunity to remove the entries here that don't exist anymore?

Reply:
done